src/native-common/AccessibilityUtil.ts

```diff
@@ -54,6 +61,32 @@ const componentTypeMap = {
 };
 
 export class AccessibilityUtil extends CommonAccessibilityUtil {
+    // Specific native platform instance for AccessibilityUtil. 
+    private _instance: PlatformAccessibilityHelpers;
+
+    constructor() {
+        super();
+
+        // Set AccessibilityUtil instance based on the platform. 
+        switch (RN.Platform.OS) {
+            case 'android':
+                this._instance = AndroidAccessibilityUtil;
```

Comment:
Instantiation of platform-specific modules should happen only in the ReactXP.ts files that are specific to each platform. You shouldn't be doing it here in this native-common module. By doing so, you're pulling in all platform implementation on all platforms. The iOS version of ReactXP shouldn't contain any Android-specific code and vice versa. By instantiating the platform-specific code in ReactXP.ts, we are able to strip out the code for other platforms. By doing it here, you defeat that mechanism.